src/DocumentationBrowserViewExtension/PackageDocumentationManager.cs

```diff
@@ -24,8 +26,11 @@ public class PackageDocumentationManager : IDisposable
         /// </summary>
         private Dictionary<string, FileSystemWatcher> markdownFileWatchers = new Dictionary<string, FileSystemWatcher>();
 
-        private const string fileExtension = "*.md";
+        private const string FILE_EXTENSION = "*.md";
```

Comment:
can you name this less generally... like MD_FILE_EXTENSION? or VALID_DOC_FILEEXTENSION